### ISSUE TITLE
Flatpak build uses EOL runtimes

### ISSUE THREAD

**Issue by Shished** [lutris/lutris#5690]
### Bug description

Lutris Flatpak build uses Gnome Runtimes (`org.gnome.Platform` and `org.gnome.Platform.Compat.i386`) version 45 which were deprecated on Sep. 18.
It needs to be rebuilt with newer runtimes because the currently used ones has stopped receiving the security updates already.

> Info: runtime org.gnome.Platform branch 45 is end-of-life, with reason:
>    The GNOME 45 runtime is no longer supported as of September 18, 2024. Please ask your application developer to migrate to a supported platform.
> Info: applications using this runtime:
>    net.lutris.Lutris
> 
> Info: runtime org.gnome.Platform.Compat.i386 branch 45 is end-of-life, with reason:
>    The GNOME 45 runtime is no longer supported as of September 18, 2024. Please ask your application developer to migrate to a supported platform.
> Info: applications using this extension:
>    net.lutris.Lutris
> 

### How to Reproduce

1. Install lutris in Flatpak: `flatpak install net.lutris.Lutris`
2. Run `flatpak update` command
3. Deprecation notice appears

### Expected behavior

Lutris should use `org.gnome.Platform` and `org.gnome.Platform.Compat.i386` Runtimes version 46 or 47.
Used runtimes are seen in the output of the `flatpak info net.lutris.Lutris` command.

### Log output

```shell
N/A (not necessary).
```

### System Information

```shell
Vulkan support: YES
Esync support: YES
Fsync support: YES
Wine installed: YES
Gamescope: YES
Mangohud: YES
Gamemode: YES
Steam: NO
In Flatpak: YES
[System]
OS: GNOME 45 Flatpak runtime
Arch: x86_64
Kernel: 6.11.5-zen1-1-zen
Desktop: GNOME
Display Server: wayland
[CPU]
Vendor: GenuineIntel
Model: 12th Gen Intel(R) Core(TM) i5-12600
Physical cores: 6
Logical cores: 12
[Memory]
RAM: 30.7 GB
Swap: 30.7 GB
[Graphics]
Vendor: AMD
OpenGL Renderer: AMD Radeon RX 6700 XT (radeonsi, navi22, LLVM 17.0.6, DRM 3.59, 6.11.5-zen1-1-zen)
OpenGL Version: 4.6 (Compatibility Profile) Mesa 24.2.5 (git-3b9fcb7e4d)
OpenGL Core: 4.6 (Core Profile) Mesa 24.2.5 (git-3b9fcb7e4d)
OpenGL ES: OpenGL ES 3.2 Mesa 24.2.5 (git-3b9fcb7e4d)
Vulkan Version: 1.3.261
Vulkan Drivers: AMD Radeon RX 6700 XT (RADV NAVI22) (1.3.289)
```

**Comment by strycore** [comment]
flathub-beta already uses the current runtime

**Comment by Shished** [comment]
It should be merged in to stable ASAP. Current runtimes are deprecated for a more than a month already.

**Comment by strycore** [comment]
meh... it will be released when we release 0.5.18.

Flathub should really stop bullying application developers into doing this sort of maintenance.
